.github/workflows/backend_plugin_test.yml

```diff
@@ -0,0 +1,58 @@
+name: Backend Plugin Testing
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+
+jobs:
+  backend_plugins:
+    runs-on: ubuntu-latest
+    env:
+      PROD_ARCHIVESSPACE_VERSION: v3.3.1
+
+    services:
+      db:
+        image: mysql:8
+        env:
+          MYSQL_ROOT_PASSWORD: root
+          MYSQL_DATABASE: archivesspace
+          MYSQL_USER: as
+          MYSQL_PASSWORD: as123
+        ports:
+          - 3307:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+    - name: Checkout ArchivesSpace
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ env.PROD_ARCHIVESSPACE_VERSION }}
+        repository: Smithsonian/archivesspace
+
+    - name: Checkout plugin
+      uses: actions/checkout@v4
+      with:
+        path: ${{ github.event.repository.name }}
+
+    - name: Copy plugin to ArchivesSpace and add to config
+      run: |
+        cp -r ${{ github.workspace }}/${{ github.event.repository.name }} ${{ github.workspace }}/plugins
+        cd ./common/config/
+        touch config.rb
+        echo "AppConfig[:plugins] = ['${{ github.event.repository.name }}']" > config.rb
+
+    - uses: Smithsonian/caas-aspace-services/.github/actions/bootstrap@main
```

Comment:
Rather than continue duplicating the same bootstrap steps in every single plugin we write/maintain, I've borrowed the newer ASpace model of moving this action into a shared action that now lives in our `aspace-services` repo: https://github.com/Smithsonian/caas-aspace-services/blob/main/.github/actions/bootstrap/action.yml

This more or less matches what's in [aspace main](https://github.com/archivesspace/archivesspace/pull/2879), and comes with the same benefits described there - caches the gems and mysql connector - an speeds up test runs.  If you look at the [workflow history on my fork](https://github.com/lorawoodford/lassb-ead-exporter/actions) you can see it dropped the runs a full minute - from about 3.5 minutes, to just under 2.5.